### PR TITLE
Added consent Menu

### PR DESCRIPTION
**Pull request by Memeji**
# Description

Added a Consent Menu to store your consent options, big thanks to 0x40 for making it and helping to get it working on floof.

---

# TODO

- [ ] Remove toggle consent example.

---

<details><summary><h1>Media</h1></summary>
<p>

Example Media Embed

</p>
</details>

---

# Changelog

:cl:
- add: Added fun :D
- tweak: Tweaked fun
- fix: Fixed fun!
- remove: Removed fun :(

### CODE REVIEW

File: Content.Shared/Floofstation/FSCVars.cs
Comment (FoxxoTrystan) [inline Content.Shared/Floofstation/FSCVars.cs:1]: Why is this file needed and not simply put in CVars?

File: Resources/keybinds.yml [hunk 1]
```diff
@@ -187,6 +187,10 @@ binds:
 - function: OpenLanguageMenu
   type: State
   key: L
+- function: OpenConsentWindow
+  type: State
+  key: C
```
Comment (FoxxoTrystan) [inline Resources/keybinds.yml:192]: Key "C" is already used, you need to change this.

File: Resources/Prototypes/Consent/examples.yml
Comment (FoxxoTrystan) [inline Resources/Prototypes/Consent/examples.yml:1]: This is just an example and reading how the code works for those toggle LOOKS HORIBLE.
Please remake fully the function on how type "ConsentToggle" is generated in the UI and server or just remove it compleatly.

**Review by FoxxoTrystan — CHANGES_REQUESTED** [review]

File: Resources/keybinds.yml (hunk 1 above)
Comment (0x6273) [reply Resources/keybinds.yml:192]: It's actually Alt+C see one line below

Comment (FoxxoTrystan) [reply Resources/keybinds.yml:192]: Correct but during testing, this reported an error, non-fetal error tho.

Comment (Fansana) [reply Resources/keybinds.yml:192]: I don't think we really need a hotkey for this. I doubt people will need to change their consent quickly.
